pkg/cmd/server/start/start_master.go

```diff
@@ -568,7 +568,7 @@ func StartAPI(oc *origin.MasterConfig, controllerPlug plug.Plug) error {
 
 	// if the webconsole is configured to be standalone, go ahead and create and run it
 	if oc.WebConsoleEnabled() && oc.WebConsoleStandalone() {
-		config, err := assetapiserver.NewAssetServerConfig(*oc.Options.AssetConfig)
+		config, err := assetapiserver.NewAssetServerConfig(*oc.Options.AssetConfig, nil)
```

Comment:
do we have a non-nil case as well? If not, drop the argument.

Reply:
> do we have a non-nil case as well? If not, drop the argument.

Asset server already moved here: https://github.com/openshift/origin-web-console-server . What's left in origin is scrap.